openmdao/test/record_util.py

```diff
@@ -0,0 +1,33 @@
+import inspect
```

Comment:
Will check on this…


On Oct 19, 2015, at 3:56 PM, Bret Naylor <notifications@github.com<mailto:notifications@github.com>>
 wrote:


In openmdao/test/record_util.py<https://github.com/OpenMDAO/OpenMDAO/pull/357#discussion_r42417772>:

> @@ -0,0 +1,33 @@
> +import inspect


We don't need this file any more, do we?

—
Reply to this email directly or view it on GitHub<https://github.com/OpenMDAO/OpenMDAO/pull/357/files#r42417772>.

---------
Herb Schilling
Scientific Applications and Visualization Team
http://ocio.grc.nasa.gov/gvis
Office of the Chief Information Officer
NASA Glenn Research Center
hschilling@nasa.gov<mailto:hschilling@nasa.gov>